program/shinyApp/R/heatmap/server.R

```diff
@@ -373,7 +373,7 @@ heatmap_server <- function(id){moduleServer(
       tmp_filename <- paste0(
         getwd(),
         file_path,
-        paste(paste0(isolate(heatmap_title()), Sys.time(), ".png"))
+        paste(paste0(isolate(heatmap_title()), Sys.Date(), ".png"))
```

Comment:
```suggestion
        paste0(isolate(heatmap_title()), Sys.Date(), ".png")
```
double paste not necessary